### DISCUSSION TITLE
Sharing the customer's timesheet with the customer possible?

### DISCUSSION THREAD

**Discussion by Tom-H-L** [kimai/kimai#3331] [Questions & Answers]
Hi,

I am wondering if it is possible to share a page with the customer where he can look up the current status of his project in the web?

For example a page where he can see:
- Hours worked on this month/period and description of tasks, and who in the team worked on it
- Current hourly budget, budget consumption and remaining budget balance

I could not figure from the docs if something like that is actually existing. Is it?
If not I would like to file it as a feature request. It is wonderful for transparency with the customers.

Best
Tom

**Comment by kevinpapst** [comment]
There is a plugin available: https://www.kimai.org/store/vettersolutions-shared-project-timesheets-bundle.html
